packaging/fedora/snap-mgmt.sh

```diff
@@ -18,7 +32,7 @@ systemctl_stop() {
     fi
 }
 
-if [ "$1" = "purge" ]; then
+purge() {
```

Comment:
I added a forum topic that we need to look into sharing code between this and the debian postrm (https://forum.snapcraft.io/t/share-code-between-debian-postrm-purge-and-snap-mgmt-sh/915)